Makefile

```diff
@@ -112,15 +112,16 @@ lint-fix: ## Fix linting issues automagically
 #	@go run -modfile tools/go.mod ./internal/lint -fix $$(go list ./... | grep -v '/acceptance/')
 	@go run -modfile tools/go.mod github.com/daixiang0/gci write -s standard -s default -s "prefix(github.com/hacbs-contract/ec-cli)" .
 
-.PHONY: tekton-lint
-tekton-lint: ## Run tekton-lint for 'tasks' subdirectory.
-ifeq ($(GITHUB_ACTIONS),) # If not running as a Github action, check if tekton-lint is installed. If not, provide link.
-	@which tekton-lint &> /dev/null || (echo "tekton-lint doesn't seem to be installed. Please see https://github.com/IBM/tekton-lint for installation instructions." && exit 1)
-endif
+node_modules: package-lock.json
```

Comment:
Missing `.PHONY` here and for `tekton-lint` below?

Reply:
Not really, they depend on `package-lock.json` timestamp and below on the `.yaml` file timestamp...